src/frontend/js/order.js

```diff
@@ -3,6 +3,35 @@ $(function() {
     // Javascript of the order application.
     // **************************************
 
+    if ($('.ui.order-override.modal')) {
+      $('.ui.order-override.modal').modal('show');
```

Comment:
@aaboffill  Ah, I misread your message.

Yes, that's true. `var $modal = $('.ui.order-override.modal')` will definitely improve the performance.